lib/features/guide/widgets/about_us/models/about_us_details.dart

```diff
@@ -0,0 +1,34 @@
+import '../../../../../api_base/directus_assets_url.dart';
+import '../../../../../utils/url_icons.dart';
+import '../../../../../utils/where_non_null_iterable.dart';
+import '../repository/about_us_repository.dart';
+import 'member_data.dart';
+
+class AboutUsDetails {
+  final AboutUs? aboutUs;
+  final List<AboutUsTeam>? aboutUsTeam;
+
+  AboutUsDetails({this.aboutUs, this.aboutUsTeam});
+
+  List<MemberData> getMemberData(List<AboutUsTeam>? aboutUsTeam) {
```

Comment:
why do you accept `List<AboutUsTeam>? aboutUsTeam` here as argument when you basically have this as class's field?